tests/utils/fixtures/fixtures.py

```diff
@@ -770,3 +770,13 @@ def commercial_test(request, bitbake_variables):
     mark = request.node.get_closest_marker("commercial")
     if mark is not None and not request.config.getoption("--commercial-tests"):
         pytest.skip("Tests of commercial features are disabled.")
+
+
+@pytest.fixture(scope="function", autouse=True)
+def cross_platform_test(request):
+    mark = request.node.get_closest_marker("cross_platform")
+    option = request.config.getoption("--cross-platform-tests")
```

Comment:
How about inverting the option (`--no-cross-platform-tests`) and passing it in mender-qa? Otherwise we might omit tests by accident, and running tests locally will skip the tests for seemingly no reason (the message is usually not displayed unless special options are passed). It also makes it more explicit that the mender-qa job is omitting certain tests, which I think is more important to know for a reader than the opposite.

Reply:
This is a good point that I have also been considering (rather, I went back and forth with it when I first implemented two selectors for "software" and "platform" tests).

With the current implementation, as you said, it is obscure that the default execution will skip a bunch of tests. With your suggestion, I then think we would need to flags: `--no-cross-platform-tests` and `--only-cross-platform-tests`. This way we can clearly and explicitly select one or the other, while still be default running all. Then just adjust mender-qa accordingly :+1: 

Once you commented on it, and I also had doubts around it, I will then go with this approach!

Reply:
Reworked now.